plugins/org.jboss.tools.jmx.remoting1_1_3/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
The idea here is to have a bundle per version of the client library we want to use. If we want to use 2.0, we will add a bundle with those libraries, not update the this one.